elastic/kibana
# [SecuritySolution] Entity store status page - Add health status to transform

## Summary

Add the actual transform status to the Entity Store status page.
Before, it just showed green whenever the transform was installed.

### How to test it?
* Install the Engine 
* Break the transform by updating it on the stack management page
* The status change should be reflected in the engine status page


![Screenshot 2024-12-16 at 11 05 30](https://github.com/user-attachments/assets/a288a037-e1ea-4747-a2bb-266300dad946)
